docs/scripts/utils.js

```diff
@@ -50,6 +51,22 @@ export function getImage() {
   return img;
 }
 
+let pieChart = null;
+
+export function getChart() {
+  if (pieChart) {
+    pieChart.update();
```

Comment:
Is this update really necesarry because you never change the chart after you made it so you can just return it?

Reply:
well... I have added at the beginning and I don't know why I left it. I'll remove it.
But I want to randomize the data of the inner chart too, when the main will be updated. In this way it will be clear that it's a chartjs instance and not an image.

Reply:
done, the randomize of inner chart data too.